common/examples.js

```diff
@@ -206,7 +215,23 @@ $(document).ready(function() {
 
 	const filterFunctionInput = $('#filter_function input');
 
-	var functionUsages = {}
+    var examplesTags = {};
+    $.getJSON('examples/examples_tags.json', function(examplesTagsData) {
+        // Filter out function usages of examples, which don't have a page
+        for (var functionName in examplesTagsData) {
+            if (findExampleByName(exampleData, functionName)) {
+                examplesTags[functionName] = examplesTagsData[functionName];
+            }
+        }
+
+        // do not handle input processing as we will pigyback (add to) the filterFunctionInput
```

Comment:
we dont want 2 event handlers for user input and i didnt want someone to think this is a mistake etc